RULEngine/Game/Field.py

```diff
@@ -88,6 +88,48 @@ def stay_outside_goal_area(self, position, is_yellow):
             position = stayOutsideCircle(position, circle_bot, self.constant["FIELD_GOAL_RADIUS"])
             return Position(position.x, position.y)
 
+    def update_field_dimensions(self, packets):
+        if not packets:
+            return
+
+        for packet in packets:
+            if packet.HasField("geometry"):
+                field = packet.geometry.field
+                self._line_width = field.line_width
+                self._field_length = field.field_length
+                self._field_width = field.field_width
+                self._boundary_width = field.boundary_width
+                self._referee_width = field.referee_width
+                self._goal_width = field.goal_width
+                self._goal_depth = field.goal_depth
+                self._goal_wall_width = field.goal_wall_width
+                self._center_circle_radius = field.center_circle_radius
+                self._defense_radius = field.defense_radius
+                self._defense_stretch = field.defense_stretch
+                self._free_kick_from_defense_dist = field.free_kick_from_defense_dist
+                self._penalty_spot_from_field_line_dist = field.penalty_spot_from_field_line_dist
+                self._penalty_line_from_spot_dist = field.penalty_line_from_spot_dist
+
```

Comment:
Do we need to keep everything the packet sends us? is it pertinenet? From these values how can we check if we received the same geometry packet as last time without having to update it all everytime.